spec/rubocop/cop/style/sole_nested_conditional_spec.rb

```diff
@@ -266,6 +266,44 @@
     RUBY
   end
 
+  context 'when the inner condition has a send node without parens' do
+    context 'in guard style' do
+      it 'registers an offense and corrects' do
+        expect_offense(<<~RUBY)
+          if foo
+            do_something if ok? bar
+                         ^^ Consider merging nested conditions into outer `if` conditions.
+          end
+        RUBY
+
+        expect_correction(<<~RUBY)
+          if foo && (ok? bar)
+            do_something
+          end
+        RUBY
+      end
+    end
```

Comment:
The following case already in parentheses seem to unneeded a new wrap. Can you add the following case?

From:

```ruby
         if foo
            do_something if ok?(bar)
                         ^^ Consider merging nested conditions into outer `if` conditions.
          end
```

To:

```ruby
          if foo && ok?(bar)
            do_something
          end
```